e2e/smartvol_ops_test.go

```diff
@@ -55,7 +56,7 @@ func testSmartVolumeDistribute(t *testing.T) {
 
 func testSmartVolumeReplicate2(t *testing.T) {
 	r := require.New(t)
-	smartvolname := "sv2"
+	smartvolname := strings.Replace(t.Name(), "/", "-", 1)
```

Comment:
Why do you need the `strings.Replace` here ?

Reply:
Test name will become ```TestSmartVolume/Smartvol_Distributed_Volume```

we cannot use / in volume names. need to replace / with -

Reply:
What might be nice is a short helper function, something like testVolName(t) to make it easier for future test writers to do the right thing.

Reply:
done